infra/cifuzz/cifuzz.py

```diff
@@ -201,6 +211,56 @@ def run_fuzzers(fuzz_seconds, workspace):
   return True, False
 
 
+def get_lastest_build_version(project_name):
+  """Gets the latest OSS-Fuzz build version for a projects fuzzers.
+
+  Args:
+    project_name: The name of the project who's build is being retrieve.
+
+  Returns:
+    A string with the latest build version or None.
+  """
+  http_get_string = GC_STORAGE + '{0}/{0}-address-latest.version'.format(
+      project_name)
+  try:
+    response = urllib.request.urlopen(http_get_string)
```

Comment:
IMO `requests` module is easier to use:

```
>>> import requests
>>> requests.get('https://storage.googleapis.com/clusterfuzz-builds/yara/yara-address-latest.version').text
'yara-address-202002140721.zip'
>>>
```

but it can also raise an exception

Reply:
I wanted to avoid having to install the extra dependency. Do you think its worth it?

Reply:
isn't `requests` available in the standard Python installation?

Reply:
Unfortunately I don't think it is. I think python currently uses urllib. It should be though. Ref: https://docs.python.org/3/library/

Reply:
It's a third party lib. I think sticking to the standard library is better for something simple like this.